packages/dds/tree/src/test/core/tree/detachedFieldIndex.spec.ts

```diff
@@ -8,22 +8,30 @@ import { strict as assert } from "node:assert";
 import type { IIdCompressor } from "@fluidframework/id-compressor";
 import { createIdCompressor } from "@fluidframework/id-compressor/internal";
 
-import { DetachedFieldIndex, type ForestRootId, RevisionTagCodec } from "../../core/index.js";
+import {
```

Comment:
The tests in tests/tree were tests for core/tree, so they have been nested into a core folder.

This was done since a new test file had to be created for core/tree/mapTree since its tests were incorrectly placed in test/simple-tree/toMapTree.spec.ts